README.md

```diff
@@ -47,7 +47,7 @@ elements, but this is not enforced by the current schema.
 
 ## License
 
-Copyright (C) 2014 The Climate Corporation. Distributed under the Apache License, Version 2.0. You may not use this library except in compliance with the License. You may obtain a copy of the License at
+Copyright © 2014-2015 The Climate Corporation. Distributed under the Apache License, Version 2.0. You may not use this library except in compliance with the License. You may obtain a copy of the License at
```

Comment:
IANAL ... But my understanding from a previous job is yes.  I'm not sure if that was an actual legal requirement to help maintain copyright, or just a corporate policy of theirs.